examples/windowmediaplayer.py

```diff
@@ -83,6 +80,22 @@ def windows_media():
     time.sleep(.5)
 
     app.ChooseColumsn.Cancel.click()
+    time.sleep(.5)
+
+    # Open a media file
+    app.WindowsMediaPlayer.menu_select("File->Open")
+
+    # Select and play a song
+    song_path = "#push your path"
+    window = app.Open
+    window.Wait('ready')

```

Comment:
I'd recommend using PEP8 complaint names: `wait` instaed of `Wait` and `click` instead of `Click`. Old names will be deprecated in 0.6.5 and removed in 0.7.0.